src/Periodicals.cpp

```diff
@@ -83,4 +83,3 @@ void every2s() {
 void every5s() {
   // Add your periodical code here
 }
-
```

Comment:
???

Reply:
I think editor removed some space? not done by me